build.gradle

```diff
@@ -396,6 +396,13 @@ ext.IS_COMPILE_HARFBUZZ = Boolean.parseBoolean(COMPILE_HARFBUZZ)
 defineProperty("COMPILE_PARFAIT", "false")
 ext.IS_COMPILE_PARFAIT = Boolean.parseBoolean(COMPILE_PARFAIT)
 
+defineProperty("STATIC_BUILD", "false")
+ext.IS_STATIC_BUILD = Boolean.parseBoolean(STATIC_BUILD)
+
+if (IS_STATIC_BUILD && IS_COMPILE_MEDIA) {
```

Comment:
Do you need to do the same for WebKit?

Reply:
No, WebKit has its own build system which we don't touch. Hence, we're making shared libs for webkit, even if STATIC_BUILD is set. A static library for webkit is probably not very practical (on iOS and Android, we use the native webkit implementations of the device).

We can modify the build to create a static build of webkit in a follow-up issue, but even when the javafx-specifc libs are static, it still makes sense to have a dynamic version of the webkit lib.

Reply:
I just figured that if that you already fail the build if you specify both `STATIC_BUILD=true` and `COMPILE_MEDIA=true` then it would make sense to also fail the build if `COMPILE_WEBKIT=true`, but as long as it doesn't break the build, then it should be OK.

Reply:
I can confirm that `gradle -PSTATIC_BUILD=true -PCOMPILE_WEBKIT=true` doesn't blow up the build on Mac, but it doesn't copy the .dylib file to the sdk/lib directory either. In any case, this can be handled as a follow-up issue.